nwbinspector/checks/nwb_containers.py

```diff
@@ -73,5 +73,5 @@ def check_empty_string_for_optional_attribute(nwb_container: NWBContainer):
     for field in fields:
         yield InspectorMessage(
             message=f'The attribute "{field}" is optional and you have supplied an empty string. Improve my omitting '
-                    f'this attribute (in MatNWB or PyNWB) or entering as None (in PyNWB)'
+            f"this attribute (in MatNWB or PyNWB) or entering as None (in PyNWB)"
```

Comment:
Minor; This level doesn't have to be an fstring to concatenate with the one above it